packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -1,9 +1,14 @@
 import nameFunction from "@babel/helper-function-name";
 import syntaxClassProperties from "@babel/plugin-syntax-class-properties";
+import { getClassIIFE } from "@babel/helper-class-iife";
+import splitExportDeclaration from "@babel/helper-split-export-declaration";
 import { template, types as t } from "@babel/core";
 
 export default function(api, options) {
-  const { loose } = options;
+  const { loose, assumePure: _assumePure } = options;
+  const assumePure = typeof _assumePure === "boolean" ? _assumePure : !!loose;
```

Comment:
I'd prefer this to throw if `assumePure` is specified and it is neither a boolean nor null/undefined.

Reply:
Gonna go with throwing on `null`s too, it doesn't give any extra meaning for the option, and one can just skip providing it (thus making it undefined). If you feel we should allow null here and treat is as undefined, please raise your concern.